benches/protobuf_convert_output.rs

```diff
@@ -0,0 +1,24 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use tailcall::blueprint::GrpcMethod;
+use tailcall::grpc::protobuf::ProtobufSet;
+
+const PROTO_FILE_PATH: &str = "src/grpc/tests/proto/greetings.proto";
+const SERVICE_NAME: &str = "greetings.Greeter.SayHello";
+
+fn benchmark_convert_output(c: &mut Criterion) {
+    let file_descriptor_set = protox::compile([PROTO_FILE_PATH], ["."]).unwrap();
+    let protobuf_set = ProtobufSet::from_proto_file(&file_descriptor_set).unwrap();
+    let method = GrpcMethod::try_from(SERVICE_NAME).unwrap();
+    let service = protobuf_set.find_service(&method).unwrap();
+    let protobuf_operation = service.find_operation(&method).unwrap();
+    let msg = b"\0\0\0\0\x0e\n\x0ctest message";
```

Comment:
instead of using string, can we use prost to serialize a struct?

Reply:
made the changes for serializing message from struct